src/lib/queries/ibc.ts

```diff
@@ -307,7 +307,7 @@ export function setupIbcExtension(base: QueryClient): IbcExtension {
             });
             channels.push(...response.channels);
             key = response.pagination?.nextKey;
-          } while (key);
+          } while (key && key.length);
```

Comment:
These are the only changes in non-autogenerated code, right?

Reply:
Right